ChangeLog.d/bugfix_PR3452.txt

```diff
@@ -0,0 +1,2 @@
+Bugfix
+   * Use local labels for fixing inline asm invalid symbol redefinition
```

Comment:
It isn't clear under what circumstances the error happens. Given the nature of the change, I think an accurate description would be:

> Use local labels in mbedtls_padlock_has_support() to fix an invalid symbol redefinition if the function is inlined.